src/Sentry.Unity/link.xml

```diff
@@ -1,5 +1,7 @@
 ﻿<linker>
     <assembly fullname="Sentry" preserve="all"/>
+    <assembly fullname="Sentry.Unity" preserve="all"/>

```

Comment:
We have this file and one on the sample project. 
Shouldn't this file be the source of truth? And somehow get added to the project at build/time?

Reply:
We should add this file via `Sentry.Unity`. For now, I included it in both places just for the **demo**, we need to think about how and when to copy it.